src/main/java/net/helpscout/api/utils/ParamsUtils.java

```diff
@@ -0,0 +1,84 @@
+package net.helpscout.api.utils;
+
+import lombok.SneakyThrows;
+
+import java.net.URLEncoder;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @Author: ivan
+ * Date: 28.04.16
+ * Time: 0:29
+ */
+public class ParamsUtils {
+
+    @SneakyThrows
+    public static String setParams(String url, Map<String, String> params) {
```

Comment:
I will check, actually, it is couple methods, so it depends on how such builder simplify method's logic, maybe it is not so painful to leave current solution.